cli/verbs.go

```diff
@@ -35,6 +37,46 @@ func run(requestor string, transport transport.Transport, stdout io.Writer) erro
 	var err error
 	switch verb {
 	case "start":
+
+		//get the environment variables
+		unparsedURL := os.Getenv("FIRMWARECI_BINURL")
+
+		//if the variable is set, download the binary
+		//the url is presumed to be from a
+		//trusted domain
+		if unparsedURL != "" {
+
+			URL, err := url.Parse(unparsedURL)
+			if err != nil {
+				return err
+			}
+
+			file, err := os.Create("/root/assets/TEST_BIN")
+			if err != nil {
+				return err
+			}
+
+			//the file name is hardcoded atm
+			defer file.Close()
+
+			switch URL.Scheme {
+			case "http":
```

Comment:
The scheme can not be https? is https supported? (it should be :) )

Reply:
It can, will fix that